spec/pronto/gitlab_spec.rb

```diff
@@ -29,6 +29,26 @@ module Pronto
           subject.should eql('prontolabs/pronto')
         end
       end
+
+      context 'http remote url' do
+        let(:repo) do
+          double(remote_urls: ['https://gitlab.example.com/prontolabs/pronto.git'])
+        end
+
+        it 'returns correct slug' do
+          subject.should eql('prontolabs/pronto')
+        end
+      end
+
+      context 'http remote url with different host' do
+        let(:repo) do
+          double(remote_urls: ['https://gitlab.example.net/prontolabs/pronto.git'])
```

Comment:
The host here is exactly the same as above, but your PR description says something different?

Reply:
@ashkulz  the host is different, here termines with `.net` and above with `.com`.

Reply:
@ArmandoAssuncao sorry, I didn't read it correctly :see_no_evil: